### ISSUE TITLE
Integrated DOI Minting Workflow

### ISSUE THREAD

**Issue by mgbilby** [scholarworks/scholarworks_v2#13]
https://groups.google.com/forum/#!topic/hydra-tech/GES2lmOdGt0

**Comment by mgbilby** [comment]
If the CSUs go with a shared DataCite membership ($8000), then we won't need to go through EZID (the UCs/CDL) to generate DOIs and can instead generate our own. EZID, to the best of my knowledge, simply resells DOIs that it gets through DataCite and charges $1/DOI. I'd imagine that other DataCite members (apparently including the UCs) have figured out how to incorporate automated DOI generation into Hydra. Perhaps we can borrow such a "gem" as well.

**Comment by scholarworks** [comment]
Yes. Any gem, or code available for this will be merged into our master. Or merged into the sufia master as part of a sprint. It's on the list :-)

**Comment by mgbilby** [comment]
Talked with Trish Cruse at DataCite a few days ago. If any one or more CSUs join DataCite as an allocating member, any and all CSUs can generate DOIs with no per item fees. She also said their developers would be happy to work on or help with integrating automated DOI generation into Hyrax/Sufia. I'm putting together a proposal in the next week or two for us at Fullerton to join DataCite, which will probably include a suggestion/invitation for other CSUs to cost share with us. I'll keep folks posted on the listserv if and when we move forward with this. But I wanted to note that DataCite developers may be able and willing to help us with this feature.
